contents/blog/equity_explainer.md

```diff
@@ -0,0 +1,99 @@
+
+1. Many of us had personally been burned by unfriendly share option schemes when working at previous companies.
+
+2. It helps us to attract talent by making our share options massively more valuable in practical terms, not just the pure $ amount (which in many ways may be the least important thing).
+
+3. It's part of our culture to expect high performance but reward people well and ensure they are genuinely able to share in PostHog's growth. Having a good share options scheme helps people feel more bought in.
+
+4. We also feel that it's just the right thing to do!
+
+## How share options work
+
+When a startup company is incorporated, the founders decide on how to divide ownership between them by (quite literally) choosing a random number of shares to create out of thin air and distributing them to each other in some proportion. 
+
+In the common two-founder model, most start-ups distribute the shares as a 50/50 split between them, accounting for 100% ownership of the business. The shares are "purchased" for a nominal cost like a ten-thousandth of a cent because the value of a newly minted start-up company is effectively nothing.
+
+As our hypothetical founders pull Red-Bull-fueled all-nighters building their MVP and applying to accelerator programs, the valuation of their company slowly increases until the point where they seek outside investors to inject capital into the business to expedite growth. 
+
+In exchange for that investment, the founders must give investors ownership in the business. But, rather than giving away their own shares, they authorize a new number of shares to be created and exchanged for real dollars. In doing so, they “dilute” their ownership by increasing the size of the total equity pie, without growing their own number of shares. 
+
+This process of dilution will continue with every new fundraising round, impacting the value of a given person’s equity as well as the founders’ authority over the business relative to their investors.
+
+Usually, the first fundraising round (the ‘seed’ round) will involve carving out a chunk of the newly generated shares to create an ‘options pool’, which can be dipped into to provide options to newly hired employees as the company grows in size. 
+
+### What exactly is an option? 
+
+It's an option to purchase a share of the company at a given price – the “strike price” – which is known as “exercising” the option, thus becoming a percent owner of the company. These options are almost always granted by way of what is known as a vesting schedule, where options become available for exercise progressively over the course of employment with the company. This is to prevent a new hire from joining a company, exercising all of their options immediately, quitting, and watching their shares in the company grow in value without contributing any work towards building the company. 
```

Comment:
```suggestion
It's an option to purchase a share of the company at a given price – the “strike price” – which is known as “exercising” the option, thus becoming a percent owner of the company. 

Options are almost always granted by way of what is known as a vesting schedule, where options become available for exercise progressively over the course of employment with the company. This is to prevent a new hire from joining a company, exercising all of their options immediately, quitting, and watching their shares in the company grow in value without contributing any work towards building the company. 
```